modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/expression/MBDecision.java

```diff
@@ -133,8 +201,17 @@ public Expression decisionAll(){
      *          [""any"", boolean, boolean, ...]: boolean
      * @return
      */
-    public Expression decisionAny(){
-        return null;
+    private Expression decisionAny() {
+        // validate the arg list
+        if (json.size() < 2) {
+            throwInsufficientArgumentCount(ALL, 1);
+        }
+        Expression[] expressions = new Expression[json.size()-1];
+        for (int i = 1; i < json.size(); ++i) {
+            Expression expression = parse.string(json, i);
+            expressions[i-1] = expression;
+        }
+        return ff.function("mbAny", expressions);
```

Comment:
This should be equivalent to a sequence of ORs

Reply:
Same as for And above